trunner/harness/plo.py

```diff
@@ -10,6 +10,20 @@
 from .base import HarnessError, IntermediateHarness, Rebooter, TerminalHarness
 
 
+@dataclass(frozen=True)
+class PloJffs2CleanmarkerSpec:
+    """Represents the JFFS2 cleanmarkers image specs to be used for formatting"""
+
+    start_block: int
+    number_of_blocks: int
+    block_size: int
+    cleanmarker_size: int
+
+    def __str__(self):
+        """Generate spec as plo jffs2 command argument"""
+        return f"{self.start_block}:{self.number_of_blocks}:{self.block_size}:{self.cleanmarker_size}"
```

Comment:
ome more thing - we might want to keep passing parameters as hex to `plo` (the logs might be more readable) - the same applies for `app` command (passed command params look strange in DEC)

```suggestion
        return f"0x{self.start_block:x}:0x{self.number_of_blocks:x}:0x{self.block_size:x}:0x{self.cleanmarker_size:x}"
```

Reply:
Yeah, it would be better, thanks for the suggestion. @maska989 could you apply those changes for other places in a separate commit?

Reply:
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/49757239/af7522d2-8a31-4663-886a-10b6c47dcf50)
I don't see it needed in the app.